Automattic/simplenote-macos
# Tracks now requires user opt-in

Changes the Privacy Settings to default to tracks being turned off by default. I added a new UserDefault that simply tracks a boolean when the user has flipped the switch. Once it has been switched, we can then enable tracks.

**To Test**
* Launch the app and view privacy settings. Verify the switch is *Off*.
* Add a breakpoint in `SPTracker` to verify that events are not sent.
* Flip the switch to on! Verify that track events start sending.
